composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
@shadowhand Gotcha. I'll do that. I didn't look closely enough at how dependencies are [resolved](https://github.com/rdlowrey/auryn#dependency-resolution).